File: solutions/uri/1005/1005.js
const input = require('fs').readFileSync('/dev/stdin', 'utf8')

const [a, b] = input
  .trim()
  .split('\n')
  .map(x => parseFloat(x))

console.log(`MEDIA = ${((a * 3.5 + b * 7.5) / 11.0).toFixed(5)}`)

File: solutions/uri/1005/1005.r
input <- file('stdin', 'r')

a <- as.double(readLines(input, n=1))
b <- as.double(readLines(input, n=1))

result = sprintf("MEDIA = %.5f", (a * 3.5 + b * 7.5) / 11.0)
write(result, "")
